UnitsNet.Tests/QuantityTests.cs

```diff
@@ -16,5 +17,42 @@ public void GetHashCodeForDifferentQuantitiesWithSameValuesAreNotEqual()
 
             Assert.NotEqual(length.GetHashCode(), area.GetHashCode());
         }
+
+        [Theory]
+        [InlineData("10 m", "9.89 m" , ComparisonType.Absolute, 0.1, false)] // +/- 0.1m absolute tolerance and some additional margin tolerate rounding errors in test case.
+        [InlineData("10 m", "9.91 m" , ComparisonType.Absolute, 0.1, true)]
+        [InlineData("10 m", "10.09 m", ComparisonType.Absolute, 0.1, true)]
+        [InlineData("10 m", "1009 cm", ComparisonType.Absolute, 0.1, true)]  // Different unit, still equal.
+        [InlineData("10 m", "10.11 m", ComparisonType.Absolute, 0.1, false)]
+        [InlineData("10 m", "8.9 m"  , ComparisonType.Absolute, 0.1, false)] // +/- 1m relative tolerance (10%) and some additional margin tolerate rounding errors in test case.
+        [InlineData("10 m", "9.1 m"  , ComparisonType.Relative, 0.1, true)]
+        [InlineData("10 m", "10.9 m" , ComparisonType.Relative, 0.1, true)]
+        [InlineData("10 m", "11.1 m" , ComparisonType.Relative, 0.1, false)]
+        public void Equals_IEquatableQuantity(string q1String, string q2String, ComparisonType comparisonType, double tolerance, bool expectedEqual)
+        {
+            IQuantity<Length, LengthUnit, double> q1 = Length.Parse(q1String, CultureInfo.InvariantCulture);
+            IQuantity<Length, LengthUnit, double> q2 = Length.Parse(q2String, CultureInfo.InvariantCulture);
+
+            Assert.Equal(expectedEqual, q1.Equals(q2, tolerance, comparisonType));
```

Comment:
Assert.NotEqual would not work with these interfaces, complaining about undefined generic members.